vignettes/lab_grading.Rmd

```diff
@@ -48,22 +48,34 @@ The DAIDS criteria can be found under the heading
 
 # Grading metadata
 
-`{admiral}` will store a metadata data set for each set of grading criteria in the data folder of 
-`{admiral}`. Currently, we have `atoxgr_criteria_ctcv4()` for NCI-CTCAEv4, `atoxgr_criteria_ctcv5()`
-for NCI-CTCAEv5 and `atoxgr_criteria_daids()` for DAIDS. Each metadata data set has required 
-variables and optional variables, the optional variables are purely for transparency, and will
-contain detailed information about the grading criteria. The required variables are those used by
-`{admiral}` to create the grade.
+`{admiral}` will store two sets of metadata data set for each set of grading criteria in the data folder of 
+`{admiral}`, one for SI units, and one for US (conventional) units. These are as follows:
+
+NCI-CTCAEv4
+  + `atoxgr_criteria_ctcv4()` for SI units.
+  + `atoxgr_criteria_ctcv4()_uscv` for US (conventional) units.
```

Comment:
is this a mistake?  `atoxgr_criteria_ctcv4()_uscv`  ??  ()_uscv?

Reply:
yeah, fixed, shouldn't have the parenthesis there anyway.